docs/components/ci-recipe-turbo-config.mdx

```diff
@@ -0,0 +1,12 @@
+```jsonc
+{
+  "$schema": "https://turbo.build/schema.json",
+  "pipeline": {
+    "build": {
+      "dependsOn": ["^build"]
+    },
+    // Add dependsOn if your tests need to run a `build` or another script first
```

Comment:
Lets add another example with the correct usage. I could see someone ending up in the same place on accident by doing 

```
    "test": {
      "dependsOn": ["^build"]
    },
```

instead of the expected

```
    "test": {
      "dependsOn": ["build"]
    },
```

Reply:
I spotted the same thing https://discord.com/channels/818588653005176832/920705046147129344/1070478043283464202

But it's not yet clear to me what is the difference between:

```
"lint": {
      "dependsOn": ["^lint"]
    },
```

and

```
"lint": {}
```